### DISCUSSION TITLE
Some recent tweets don't appear in subscription

### DISCUSSION THREAD

**Discussion by Piscium** [j-fbriere/squawker#58] [Q&A]
First of all, many thanks to the developers of squawker and its predecessors for that time and effort to provide twitter clients!

I came across an 'issue', however I am not sure it is really an issue so I am starting this discussion instead of filing a bug report, it could be that I am not using squawker well.

In squawker I added a subscription to:
https://twitter.com/DublinTango

In squawker the last post I see was 6 years ago. However there are two recent posts, this is the most recent:
https://twitter.com/DublinTango/status/1688501292929282048

So my question is, why does this recent post not appear in my squawker subscription? Am I doing something wrong? The same thing used to happen with fritter (when it worked). It could be that the twitter algorithms are not providing that data to API clients or something. I am using squawker v. 3.4.2. Also I noticed that this sort of thing happens with another subscription I have, it shows old posts but not recent ones, so it is not exclusive to the twitter handle I gave as an example above.

**Comment by ghost** [comment]
Does this still happen on the latest version of squawker?

**Reply by Piscium** [reply]
It still happens on 3.4.4. The latest version, 3.5.0, has only two bug fixes that seem unrelated to my issue so I have not upgraded to 3.5.0 as the last time I installed a new version I could not see tweets for a while, I think because of guest keys being blocked by Twitter or something.

**Comment by Piscium** [comment]
I am closing this as it is not relevant anymore given such an old version of Squawker and all the changes in Twitter.
